addons/website_slides/views/website_slides_templates.xml

```diff
@@ -148,247 +64,322 @@
-                                    <t t-set="hash" t-value="message_post_hash"/>
-                                    <t t-set="pid" t-value="message_post_pid"/>
-                                    <t t-set="default_message_id" t-value="last_message_id"/>
-                                    <t t-set="default_message" t-value="last_message"/>
-                                    <t t-set="default_rating_value" t-value="last_rating_value"/>
-                                    <t t-set="force_submit_url" t-value="'/slides/mail/update_comment' if last_message_id else False"/>
-                                </t>
-                            </p>
-                            <p class="row mt8">
-                                <b>Share:</b>
-                                <t t-call="website_slides.slides_share">
-                                    <t t-set="slide" t-value="channel.slide_ids[0]"/>
-                                </t>
-                            </p>
                         </div>
                     </div>
                 </div>
-            </section>
-            <section class="o_slides_discussion_rating">
-                <div class="container mt16 mb16">
-                    <hr/>
-                    <div class="row">
-                        <div class="col-lg-8 offset-lg-2">
-                            <t t-call="portal.message_thread">
-                                <t t-set="object" t-value="channel"/>
-                                <t t-set="display_rating" t-value="True"/>
-                                <t t-set="disable_composer" t-value="True"/>
-                            </t>
+            </div>
+            <div class="row o_wslides_course_content">
+                <div class="col-lg-12 o_wslides_container o_wslides_content_container">
+                    <div class="row justify-content-end o_wslides_course_content_nav">
+                        <div class="col-8">
+                            <!-- nav-tabs-border -->
+                            <ul class="nav nav-tabs o_wslides_nav_tabs" role="tablist" id="profile_extra_info_tablist">
+                                <!-- <li class="nav-item" id="profile_nav_tab_about">
+                                    <a role="tab" aria-controls="about" href="#profile_tab_content_about" class="nav-link o_wprofile_navlink active" data-toggle="tab">About</a>
+                                </li> -->
+                                <li class="nav-item">
+                                    <a t-att-href="'/slides/%s%s?%s' % (slug(channel), '/%s' % slug(category) if category else '', keep_query('debug'))"
+                                        t-att-class="'nav-link o_wslides_navlink %s' % ('' if search_slide_type else 'active')">
+                                        <i class="fa fa-home"/> Home</a>
+                                </li>
+                                <li class="nav-item">
```

Comment:
Is it intended that both Form and Review lands on the same url ?

Reply:
Well waiting for other tasks: forum -> forum integration, and review -> allow comments.